Neos.Eel/Classes/FlowQuery/FlowQuery.php

```diff
@@ -120,6 +120,17 @@ public function __construct($context, array $operations = [])
         $this->operations = $operations;
     }
 
+    /**
+     * The q function from eel
+     * @param mixed
+     * @return FlowQuery
+     * @throws Exception
+     */
+    public static function q($element)
```

Comment:
```suggestion
    public static function q($element): self
```
return type annotation?